fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,24 @@
+
+#header{
+    display: flex;
+    justify-content: space-between;
+}
+.right{
+    display: flex;
+    list-style: none;
+}
+
+#header-img{
+    width: 100%;
+    height: 100%;
+    max-width: 400px;
+}
+#submit{
```

Comment:
I think there should be spaces between the css element name and the opening curly braces. Using linters for html & css file would make it easier. Check out this link https://github.com/microverseinc/linters-config/tree/master/html-css